encoders/src/test/scala/sql/EncoderDerivationSpec.scala

```diff
@@ -235,7 +234,11 @@ class EncoderDerivationSpec extends munit.FunSuite with SparkSqlTesting:
     "derive encoder of case class F"
   ) {
     val encoder = summon[Encoder[F]]
-    val input = Seq(F(Some("whoo"), None), F(None, Some(1L)), F(null, null))
+    val input = Seq(
```

Comment:
I had some difficulties wrapping tuples with UDFs - hence some small changes (also the nullable) and this test